tools/java/java-build/test/com/google/i18n/phonenumbers/BuildShortNumberMetadataJsonFromXmlGoldenTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2018 The Libphonenumber Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.i18n.phonenumbers;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStreamReader;
+import junit.framework.TestCase;
+
+/**
+ * Tests for the output of the JSON metadata producer.
+ */
+public final class BuildShortNumberMetadataJsonFromXmlGoldenTest extends TestCase {
+
+  private static final String INPUT_FILE_NAME = "ShortNumberMetadataForGoldenTests.xml";
```

Comment:
Why make INPUT_FILE_NAME a constant but not expected_shortnumbermetadata.js? I'd make that a constant too.
E.g.
GOLDEN_INPUT_FILE_NAME
GOLDEN_OUTPUT_FILE_NAME

Reply:
sgtm. Done for both the regular and short number metadata golden test.